index.js

```diff
@@ -1,4 +1,23 @@
-(function() {
+(function(factory) {
+
+  if (typeof exports !== 'undefined') {
+    var _factory = factory();
+    module.exports = _factory;
+    module.exports.Factory = _factory.Factory;
+    module.exports.Adapter = _factory.Adapter;
+    module.exports.ObjectAdapter = require('./lib/object-adapter');
+  }
+  else if (typeof define === 'function' && define.amd) {
+    define(['factory-girl-object-adapter'], function(ObjectAdapter) {
+      var _factory = factory();
+      _factory.ObjectAdapter = ObjectAdapter;
+      return _factory;
+    });
+  }
+  else {
+    factory();
+  }
```

Comment:
Sorry, I forgot to attach to window to use on browser. Feel free to remove if you don't want to expose this functionality.
I made another change to fix that (commit: 6bc7c44)